lib/include/tick/prox/prox_l1w.h

```diff
@@ -49,6 +56,22 @@ class DLL_PUBLIC TProxL1w : public TProxSeparable<T> {
 
   void set_weights(SArrayTPtr weights) { this->weights = weights; }
 
+  template <class Archive>
+  void serialize(Archive &ar) {
+    ar(cereal::make_nvp("ProxSeparable",
+                        cereal::base_class<TProxSeparable<T>>(this)));
+
+    ar(CEREAL_NVP(weights));
```

Comment:
So does this one pass the gtests? Is so I think we can merge :)

Reply:
I have tested it on a future branch with solvers and yes it does

Reply:
Alright, let's merge then !